src/assets/test_data/US_ND_counties.json

```diff
@@ -0,0 +1,320 @@
+[
```

Comment:
Because this is a one-off file I took the following liberties to make it easier to work with this data:

- I removed the "US_ND_" prefix from each of the county codes (i.e. "US_ND_ADAMS" becomes "ADAMS").  This made it easier/cleaner to map the counties in `assets/maps/us_nd.json` to Program Regions and Judicial Districts.
- I removed all of the judicial_district_codes that were marked as "OUT_OF_STATE"

Strictly speaking, we're not using this file anywhere but since I took the time to clean it up, I figured I'd include it in case we need it in the future.